layers/kubernetes-master-worker-base.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "kubernetes-master-worker-base",
+  "name": "kubernetes-master-worker-base",
+  "repo": "https://github.com/juju-solutions/kubernetes.git",
```

Comment:
We now support an explicit `docs` field in the JSON, so I'd like to see it added to point to https://github.com/juju-solutions/kubernetes/blob/master/cluster/juju/layers/kubernetes-master-worker-base/README.md directly, rather than the general K8s repo README.